lib/Itho/IthoCC1101.cpp

```diff
@@ -626,11 +754,85 @@ const uint8_t * IthoCC1101::getMessageCommandBytes(IthoCommand command)
       return &ithoMessageJoinCommandBytes[0];
     case IthoLeave:
       return &ithoMessageLeaveCommandBytes[0];
+
+    case OrconStandBy:
+      return &orconMessageStandByCommandBytes[0];
+    case OrconLow:
+      return &orconMessageLowCommandBytes[0];
+    case OrconMedium:
+      return &orconMessageMediumCommandBytes[0];
+    case OrconHigh:
+      return &orconMessageFullCommandBytes[0];
+    case OrconAuto:
+      return &orconMessageAutoCommandBytes[0];
+    case OrconTimer0:
+      return &orconMessageTimer0CommandBytes[0];
+    case OrconTimer1:
+      return &orconMessageTimer1CommandBytes[0];
+    case OrconTimer2:
+      return &orconMessageTimer2CommandBytes[0];
+    case OrconTimer3:
+      return &orconMessageTimer3CommandBytes[0];
+    case OrconAutoCO2:
+      return &orconMessageAutoCO2CommandBytes[0];
+
     default:
       return &ithoMessageLowCommandBytes[0];
   }
 }
 
+uint8_t IthoCC1101::getMessageCommandLength(IthoCommand command)
+{
+  switch (command)
+  {
+    case IthoStandby:
+      return sizeof(ithoMessageStandByCommandBytes) / sizeof(uint8_t);
```

Comment:
Not sure if this will be correct.
Please check as I have no idea whether you're checking for the array length or the `sizeof(pointer)`

Reply:
Checked a few different sized arrays and it returns the expected size (length of the command bytes array).
Also checked what it could save when replaced by constant values (case with fall-through), but that's already optimized by the compiler (16 bytes gained on ESP32, none on ESP8266), so no reason to bother about that.

Reply:
Hmm that surprises me, since switch statements aren't that "cheap" regarding bin size and also I don't know whether the compiler will already compute the return value at compile time, or that we might need to add `constexpr` to the code to tell the compiler that it can already compute the value.